src/shaders/symbol_icon.vertex.glsl

```diff
@@ -29,6 +32,13 @@ varying float v_fade_opacity;
 
 #pragma mapbox: define lowp float opacity
 
+vec2 absfract(vec2 value) {
+    vec2 result = fract(value);
+    if (result.x > 0.5) result.x -= 1.0;
+    if (result.y > 0.5) result.y -= 1.0;
+    return result;
```

Comment:
Shorter way to write it `return fract(value) - vec2(greaterThan(color, vec2(0.5)));`. Shouldn't matter, though.

Reply:
I can try, but many OpenGL implementations I tested are _very_ picky about subtle changes like this and perform rounding differently.